### DISCUSSION TITLE
Looking for offline Map view and offline navigation support in my android applicatiton.

### DISCUSSION THREAD

**Discussion by vidhyablrgithub** [osmdroid/osmdroid#1926] [Q&A]
Can this library be used for offline view and tracking the routes on latest Android platforms?
Do we have sample app developed for the same on the latest Android versions?

**Comment by spyhunter99** [comment]
view offline tiles? sure
routing offline? no. For routing, you need an online service to get at least get the route. Try osmbonuspack for routing tools
